about/events/2019-05-17-maker-faire-berlin.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: Event
+category: event
+title: Maker Faire Berlin
+organizer: Maker Media GmbH
+link: 'https://maker-faire.de/berlin/'
+date: '2018-05-17T23:49:15+02:00'
```

Comment:
Are you living in the past @github-luckow-org? :laughing:

Reply:
argh. template from last year :)